### PR TITLE
Update raspberry_pi_installation.md

### PR DESCRIPTION
**Pull request by MatsA**
Added info about the "Data browser", corrected a link to RPi access point, added info about the default NMEA 0183 output and some fine tuning....

### CODE REVIEW

File: raspberry_pi_installation.md [hunk 1]
```diff
@@ -213,8 +220,14 @@ Or the path to NMEA0183 file
 
 ## NMEA0183 data on the network
 
-Many navigation or marine data applications just accept NMEA0183 data as input. SignalK have a plugin which can forward this data, to Your Tablet or Phone, and being processed in an application like iNavX, iSailor, SeaPilot and so on.
-Attach the SignalK server to a network with WiFi access, or set up the RPi as an access point(link below), and configure the signalk-to-nmea0183 plugin.
+Many navigation or marine data applications, could be iNavX, iSailor, SeaPilot....., just accepts NMEA 0183 data as input. The SignalK servers default output for NMEA0183 data is port 10 110 using the TCP protocol. Any NMEA0183 input to the SignalK server is forwarded to this output. If You are using a none NMEA 0183 input, could be NMEA 2000, You have to use a plugin that converts the data to NMEA 0183.
```
Comment (sbender9) [inline raspberry_pi_installation.md]: There's an extra space in the port: "10 110"

Comment (MatsA) [reply raspberry_pi_installation.md]: OK, changed..... Been in the banking business to long....  ;)

Comment (tkurki) [reply raspberry_pi_installation.md]: _could be iNavX, iSailor, SeaPilot.....,_ => _for example iNavX, iSailor, SeaPilot,_

_just accepts_ => _accept only_

_servers_ => _server's_ 

I'd use lowercase _you_.

Comment (MatsA) [reply raspberry_pi_installation.md]: OK, updated and added link to FAQ

**Pull request merged**